src/components/CheckList/CheckListScene.tsx

```diff
@@ -0,0 +1,196 @@
+  SceneDataTransformer,
+  SceneFlexItem,
+  SceneFlexLayout,
+  SceneQueryRunner,
+  SceneReactObject,
+  SceneRefreshPicker,
+  SceneTimePicker,
+  SceneTimeRange,
+  VariableValueSelectors,
+  VizPanel,
+} from '@grafana/scenes';
+import { Spinner } from '@grafana/ui';
+import { InstanceContext } from 'contexts/InstanceContext';
+import React, { useContext, useMemo } from 'react';
+import { CheckListViewType, DashboardSceneAppConfig } from 'types';
+import { CheckListViewSwitcher } from './CheckListViewSwitcher';
+import { AddNewCheckButton } from './AddNewCheckButton';
+
+function getCheckListScene(config: DashboardSceneAppConfig & Props) {
+  const queryRunner = new SceneQueryRunner({
+    datasource: config.metrics,
+    queries: [
+      {
+        editorMode: 'code',
+        expr: `sum by (instance, job, check_name) (
+              rate(probe_all_success_sum[$__range])
+              * 
+              on (instance, job)
+              group_left(check_name)
+              max by (instance, job, check_name)
+              (sm_check_info)
+        ) 
+        /
+        sum by (instance, job, check_name) (
+              rate(probe_all_success_count[$__range])
+              * 
+              on (instance, job)
+              group_left(check_name)
+              max by (instance, job, check_name)
+              (sm_check_info)
+        ) 
+        `,
+        legendFormat: '{{job}}',
+        format: 'table',
+        range: false,
+        instant: true,
+        refId: 'A',
+      },
+    ],
+  });
+
+  const transformed = new SceneDataTransformer({
+    $data: queryRunner,
+    transformations: [
+      {
+        id: 'sortBy',
+        options: {
+          sort: [
+            {
+              desc: false,
+              field: 'Value',
+            },
+          ],
+        },
+      },
+    ],
+  });
+
+  const timeRange = new SceneTimeRange({
+    to: 'now',
+    from: 'now - 6h',
+  });
+
+  return new EmbeddedScene({
+    $timeRange: timeRange,
+    controls: [
+      new SceneReactObject({
+        //@ts-ignore not sure what's going on here, but the types are unhappy for some reason
+        component: CheckListViewSwitcher,
+        props: {
+          viewType: CheckListViewType.Viz,
+          setViewType: config.setViewType,
+          setCurrentPage: config.setCurrentPage,
+        },
+      }),
+      new VariableValueSelectors({}),
+      // new CustomOb
+      new SceneControlsSpacer(),
+
+      // customObject,
+      new SceneTimePicker({ isOnCanvas: true }),
+      new SceneRefreshPicker({
+        intervals: ['1m', '5m', '15m', '1h'],
+        isOnCanvas: true,
+      }),
+      new SceneReactObject({
+        component: AddNewCheckButton,
+      }),
+    ],
+    body: new SceneFlexLayout({
+      children: [
+        new SceneFlexItem({
+          width: '100%',
+          height: '100%',
+          body: new VizPanel({
+            pluginId: 'stat',
+            title: '',
+            $data: transformed,
```

Comment:
Agree with @torkelo that using the repeater here and showing it in a different way than stat tiles may be better experience. Also this will allow you to control `minHeight` of each individual viz making it behave better on mobile devices (not rish of an element getting completely squshed)

Reply:
I think this stat panel view is only aiming to replace this honeycomb view. But for the main checks list, a repeater that repeats a layout is definitely the solution.

![image](https://github.com/grafana/synthetic-monitoring-app/assets/10999/b95e5198-49df-41dc-8d18-8e37e915186f)